features/foutafhandeling.feature

```diff
@@ -5,141 +5,65 @@ Functionaliteit: Afhandeling van fouten
-        | title  | Een of meerdere parameters zijn niet correct. |
-        | status | 400                                           |
-        | code   | paramsValidation                              |
-        | detail | Bad request.                                  |
-        En bevat de response de volgende invalidParams
-        | name   | code   | reason                                                    |
-        | expand | expand | Deel van de parameterwaarde is niet correct: bestaatniet. |
+Rule: min waarde is niet groter dan max waarde
 
-    Scenario: Er zijn meerdere onbekende resource namen opgegeven
-        Als '/adressen?zoekresultaatIdentificatie=adr-f6b00930705c1ef12c2624623a797d21&expand=bestaatniet,bestaatookniet' wordt aangeroepen
+    Abstract Scenario: opgegeven minimale oppervlakte is groter dan opgegeven maximale oppervlakte
+        Als '<path><query string>' wordt aangeroepen
         Dan bevat de response de volgende velden
         | naam   | waarde                                        |
         | title  | Een of meerdere parameters zijn niet correct. |
         | status | 400                                           |
         | code   | paramsValidation                              |
         | detail | Bad request.                                  |
         En bevat de response de volgende invalidParams
-        | name   | code   | reason                                                       |
-        | expand | expand | Deel van de parameterwaarde is niet correct: bestaatniet.    |
-        | expand | expand | Deel van de parameterwaarde is niet correct: bestaatookniet. |
+        | name             | code   | reason                           |
+        | oppervlakte[min] | range  | min mag niet hoger zijn dan max. |
 
-    Scenario: Er is één of meerdere onbekende veld namen voor één of meerdere resource opgegeven
-        Als '/adressen?zoekresultaatIdentificatie=adr-f6b00930705c1ef12c2624623a797d21&expand=nummeraanduiding.bestaatniet' wordt aangeroepen
-        Dan bevat de response de volgende velden
-        | naam   | waarde                                        |
-        | title  | Een of meerdere parameters zijn niet correct. |
-        | status | 400                                           |
-        | code   | paramsValidation                              |
-        | detail | Bad request.                                  |
-        En bevat de response de volgende invalidParams
-        | name   | code   | reason                                                                     |
-        | expand | expand | Deel van de parameterwaarde is niet correct: nummeraanduiding.bestaatniet. |
+        Voorbeelden:
+        | path                   | query string                               | code  | reason                           |
+        | /adresseerbareobjecten | ?oppervlakte[min]=200&oppervlakte[max]=100 | range | min mag niet hoger zijn dan max. |
```

Comment:
dit is een abstract scenario met 1 voorbeeld...

Waarom niet ook een voorbeeld met bouwjaar?